client/src/components/formHelpers/FormFields.js

```diff
@@ -10,8 +10,24 @@ import {
   HelpBlock
 } from '@freecodecamp/react-bootstrap';
 import { Field } from 'react-final-form';
+/* eslint-disable-next-line max-len */
+import { SolutionFormContext } from '../../templates/Challenges/projects/SolutionForm';
+import {
+  editorValidator,
+  localhostValidator,
+  composeValidators
+} from './FormValidators';
 
 const propTypes = {
+  errors: PropTypes.arrayOf(
+    PropTypes.shape({
+      error: PropTypes.shape({
+        message: PropTypes.string,
+        stack: PropTypes.string
+      }),
+      value: PropTypes.string
+    })
+  ),
```

Comment:
```suggestion
```

Reply:
Is there a better way to represent the shape of this data?